src/libs/actions/Report.js

```diff
@@ -54,6 +56,35 @@ Onyx.connect({
     },
 });
 
+/**
+ * Map of the most recent non-loading report history items we have stored for a report. Not to be confused with the
+ * reportMaxSequenceNumbers. We can compare these values to those to determine whether there is "available history"
```

Comment:
I'm not sure what "these" and "those" refer to. Can you update this to be explicit, please?